src/persist/src/indexed/cache.rs

```diff
@@ -68,6 +68,7 @@ impl<B> Clone for BlobCache<B> {
 
 impl<B: BlobRead> BlobCache<B> {
     const META_KEY: &'static str = "META";
+    const DEFAULT_CACHE_SIZE_LIMIT: usize = 2_000_000_000;
```

Comment:
I have a strong preference for plumbing this all the way to a cli flag in this PR. as soon as we have a cache, we're going to want to be able to tune it without recompiling

concretely: it should be a single flag with a reasonable default: the 4GiB you have here sounds fine to me. that single number should make it all the way to persist, which then splits it as necessary (initially, the heuristic being 50/50)

nit: make the default a power of 2, not a round base-10 number :)

Reply:
done to both. just to be clear though the default here is 2 GiB which gets split up between unsealed and trace. you can see that in `BlobCacheInner::new`